dangerfile.js

```diff
@@ -0,0 +1,29 @@
+import { danger, warn } from 'danger'
```

Comment:
this is a staring place for automating helpful things do make prs easier to manage. thoughts? suggestions?